glusterd2/volgen/store-utils.go

```diff
@@ -0,0 +1,91 @@
+package volgen
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+
+	"github.com/gluster/glusterd2/glusterd2/store"
+	"github.com/gluster/glusterd2/pkg/errors"
+
+	"github.com/coreos/etcd/clientv3"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	volfilePrefix = "volfiles/"
+)
+
+type volfileData struct {
+	VolumeName string
+	Content    string
+}
+
+func save(name string, content string, volname string) error {
+	jsondata, err := json.Marshal(volfileData{VolumeName: volname, Content: content})
+	if err != nil {
+		log.WithField("error", err).Error("Failed to marshal the volfile data")
+		return err
+	}
+	_, err = store.Put(context.TODO(), volfilePrefix+name, string(jsondata))
+	return err
+}
+
+// GetVolfiles returns list of all Volfiles
+func GetVolfiles() ([]string, error) {
+	resp, e := store.Get(context.TODO(), volfilePrefix, clientv3.WithPrefix())
+	if e != nil {
+		return nil, e
+	}
+
+	volfiles := make([]string, len(resp.Kvs))
+
+	for i, kv := range resp.Kvs {
+		volFile := strings.TrimPrefix(string(kv.Key), "volfiles/")
+		volfiles[i] = volFile
+	}
+
+	return volfiles, nil
+}
+
+//GetVolfile return particular volfile info
+func GetVolfile(volfileID string) ([]byte, error) {
+	volfile := volfilePrefix + volfileID
```

Comment:
nit: Please use `err` as variable name if possible. I understand that this code may have come from somewhere else where it's used as `e`

Reply:
@prashanthpai I think using short names is better if the variable is used very close to its declaration/initialization. Just read it https://talks.golang.org/2014/names.slide#4. If its a convention in our source code then its better to use error.